content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md

```diff
@@ -5,63 +5,77 @@ title: Add a new VPN user in the MyRackspace Portal
 
-   b. If you need to add the VPN user to all firewalls on the account, ensure
+**Step 5.** In the **Issue Details** section you can select one, multiple or all of the following VPN user management options as shown in following image.
+
+   <img width="633" alt="Select any one or multiple VPN user on firewall" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn5.png">
+
+**Step 6.** Check the box next to **Create new VPN user on firewall**.
+
+   <img width="408" alt="Select the create new VPN option" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn6.png">
+
+**Step 7.** In the **Firewall** drop-down field, select the appropriate firewall(s).
+
+   <img width="370" alt="Select required firewall from the drop-down list" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn7.png">
+
+   a. If you need to add the VPN user to all firewalls on the account, ensure
       you select all firewalls in this field.
 
-7. Enter your desired VPN username in the **VPN Username** field.
+**Step 8.** Enter your desired VPN username in the **VPN Username** field.
+
+   <img width="374" alt="Enter VPN username" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn8.png">
 
    a. The VPN username is case-sensitive and requires at least three characters.
 
    b. Do not include a question mark (?) or space ( ) within the VPN username field.
 
-8. The **Generate Random Password** box shows as checked by default. If you want
-   to add a custom password, uncheck this box.
+**Step 9.** The **Generate Random Password** box shows as checked by default. 
 
-   After you uncheck the box, the custom password field displays.
+   <img width="299" alt="By default Generate Random Password is selected" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn10.png">
+
+   a. If you want to add a custom password, uncheck the box **Generate Random Password**. After you uncheck the box, the custom password field displays.
 
    This password must have at least eight characters with uppercase, lowercase,
    numbers, and special characters.
 
-9. Click **Create Ticket** at the bottom of the page.
+   <img width="346" alt="Deselect Generate Random Password to generate your password" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn9.png">   
+
+**Step 10.** Click **Submit** at the bottom of the page.
 
    After the ticket generates, you see your ticket list in
    the MyRackspace Portal. Automation runs in the background while the ticket
    generates, which typically takes less than a minute to complete. After you
    refresh your browser, the ticket, *Create New VPN User on Firewall Request (from template)*,
    is in **Confirm Solved** status.
 
-10. Retrieve the password for your new VPN user by opening the ticket. The
+**Step 11.** Retrieve the password for your new VPN user by opening the ticket. The
     password is in the most recent comment.
 
    {{<image src="image002.jpeg" alt="" title="">}}
```

Comment:
I was about to merge, but it seems this image needs to be updated. Looks like an old image that doesn't relate to this step. Please review and fix.
I applied commits to this PR. So you will need to checkout this PR branch on your local and do git pull first. then make your commits and push back up again.

Reply:
Replaced the image with the new one.